avalanche/evaluation/metrics/accuracy.py

```diff
@@ -379,6 +445,7 @@ def accuracy_metrics(
 
 __all__ = [
     "Accuracy",
+    "TaskAwareAccuracy",
```

Comment:
Shouldn't the `*_metrics` functions exposes the `split_by_task` option? This would make it easier to get the corresponding metrics.

Reply:
the `*_metrics` just provide reasonable defaults.